.gitlab-ci.yml

```diff
@@ -13,8 +13,10 @@ stages:
 variables:
   GIT_STRATEGY:                    fetch
   CARGO_HOME:                      "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_JOB_NAME}"
+  # CARGO_TARGET_DIR:                "/ci-cache/${CI_PROJECT_NAME}/target/${CI_JOB_NAME}"
```

Comment:
Please remove instead of commenting out. We have git history anyway to restore if needed.